protoc-gen-hack/plugin.go

```diff
@@ -29,7 +30,17 @@ var (
 	version          = "undefined" // go build -ldflags "-X main.version=1"
 	fversion         = flag.Bool("version", false, "print version and exit")
 	reservedKeywords = []string{"eval", "isset", "unset", "empty", "const", "new", "and", "or",
-		"xor", "as", "print", "throw", "array", "instanceof", "trait", "class", "interface", "static", "self"}
+		"xor", "as", "print", "throw", "array", "instanceof", "trait", "class", "interface", "static", "self",
```

Comment:
I think either the words in this list should be lowercased or they should be lowercased before comparison here:

https://github.com/y3llowcake/proto-hack/blob/952fe37c7f357877c27e7d2be82006e4a9e0f4fb/protoc-gen-hack/plugin.go#L246

Reply:
good catch, I've updated the PR but lowercasing keywords on line 246